internal/impl/kafka/output_kafka_franz.go

```diff
@@ -111,14 +110,12 @@ type franzKafkaWriter struct {
 
 	client *kgo.Client
 
-	log     *service.Logger
-	shutSig *shutdown.Signaller
```

Comment:
It's not used in this component, but should it? I see that inputs do leverage it in the disconnect / shutdown logic

Reply:
We definitely don't need a shutdown signaller, that's probably a copy/paste artifact.